src/dbt/kipptaf/models/extracts/tableau/intermediate/int_tableau__student_enrollments.sql

```diff
@@ -0,0 +1,101 @@
+with
+    nj_state_assessment_demo as (
+        select distinct
+            _dbt_source_relation,
+            academic_year,
+            statestudentidentifier as state_id,
+
+            coalesce(studentwithdisabilities in ('504', 'B'), false) as is_504,
+
+            if(englishlearnerel = 'Y', true, false) as lep_status,
+
+            case
+                when twoormoreraces = 'Y'
+                then 'T'
+                when hispanicorlatinoethnicity = 'Y'
+                then 'H'
+                when americanindianoralaskanative = 'Y'
+                then 'I'
+                when asian = 'Y'
+                then 'A'
+                when blackorafricanamerican = 'Y'
+                then 'B'
+                when nativehawaiianorotherpacificislander = 'Y'
+                then 'P'
+                when white = 'Y'
+                then 'W'
+            end as race_ethnicity,
+
+            case
+                when studentwithdisabilities in ('IEP', 'B')
+                then 'Has IEP'
+                else 'No IEP'
+            end as iep_status,
+
+        from {{ ref("int_pearson__all_assessments") }}
+        where academic_year >= {{ var("current_academic_year") - 7 }}
+    ),
+
+    students as (

```

Comment:
you can remove this CTE

Reply:
If I remove it and replace nj_state_assessment_demo with {{ ref("int_pearson__all_assessments") }}, are you OK with me using distinct on the main select statement? Otherwise, I will have dups.

Reply:
you're not using distinct on this currently. I'm just saying, you can take it out of the CTE

Reply:
I was. I removed it when I thought I understood what you meant and didnt put it back.